modules/openxr/extensions/platform/openxr_opengl_extension.cpp

```diff
@@ -39,6 +39,8 @@
 #include "servers/rendering/rendering_server_globals.h"
 #include "servers/rendering_server.h"
 
+#define GRAPHICS_API_NAME "OpenGL"
+
```

Comment:
I don't know about using a `#define` for this. It'd probably be better to just use the string right in the function, or if anything maybe `constexpr` rather than `#define`?

Reply:
I'm not sure what we're trying to do here or where this is used, but this information is already available from the rendering configuration.

I'm not a fan of having the OpenXR graphics extensions accessible, the whole idea is that they are always accessed through OpenXRAPI so that things remain graphics driver agnostic as much as possible.

Reply:
We could rely on the info from the rendering configuration. However, assuming we expose `OpenXRAPIExtension::get_supported_swapchain_formats()` (in this PR), which gives us ids specific to a rendering API, then I think it makes sense for `OpenXRAPIExtension` to also be the one to tell us which rendering API those ids are specific to.

> I'm not a fan of having the OpenXR graphics extensions accessible, the whole idea is that they are always accessed through OpenXRAPI so that things remain graphics driver agnostic as much as possible.

This isn't directly exposing the graphics extensions. We'd still be using `OpenXRAPIExtension` to interact with them, via the `OpenXRAPIExtension::openxr_swapchain_*()` functions, and `OpenXRAPIExtension::get_supported_swapchain_formats()` and `OpenXRAPIExtension::get_graphics_api_name()` (which is what this specific change is about).

Reply:
In the latest push I've updated the functions themselves to just return the strings. I would agree with David that it makes sense to provide the API name alongside the swapchain formats in `OpenXRAPIExtension`.